src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs

```diff
@@ -8,9 +8,11 @@
 using Diagnostics.RuntimeHost.Models;
 using Diagnostics.RuntimeHost.Services;
 using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Diagnostics.RuntimeHost.Controllers
 {
+    [Authorize]
     public abstract class SiteControllerBase : DiagnosticControllerBase<App>
```

Comment:
We add the tag to Abstract base class as well? Will this work?

Reply:
You also have authorize attribute on the `SitesController` class so I assume you won't need this here.